docs/repo-docs/getting-started/support-policy.mdx

```diff
@@ -69,3 +69,52 @@ We highly encourage you to use the most current version of Turborepo whenever po
 | ------------- | ------------ | ---------------- |
 | 2.x           | Jun. 4, 2024 |                  |
 | 1.x           | Dec. 9, 2021 | Jun. 4, 2026     |
+
+## Release phases
+
+Stable builds of Turborepo follow semantic versioning (semver) and are thoroughly tested before being released.
+
+Turborepo also offers specific APIs that may be released in an earlier phase of development. This is to allow early adopters to give valuable feedback and help the core team gain confidence in APIs ahead of their stable release. APIs that are available pre-stable will be marked as such in the documentation and within `turbo` itself whenever possible.
+
+Not every API will be released in every phase. We use our best judgement to determine the phase for an API based on our confidence for the feature, weighing factors like the feature's complexity, possible use cases, and feedback from early users of the API.
+
+The Turborepo core team would like to thank any users who do use features early and express our appreciation for any feedback you provide.
+
+### Stable
+
+When an API reaches stability, it is considered ready for use in production. APIs marked as stable will only change in a backward-compatible way in a major release.
```

Comment:
Do we want to clarify that we do not consider unstructured terminal output as an API? For example https://github.com/vercel/turborepo/pull/10079 should not be considered a breaking change IMO. Now if it was a removal of a field from `turbo ls --output=json` it would be.

Reply:
My gut reaction is to say no, but I could be talked into saying yes. This has come up _once_ in my memory, or has it been more?

Maybe a broader question: Do other CLI tools consider their stdout as semver-protected? I've never looked into or thought about this...

Reply:
Relevant xkcd: https://xkcd.com/1172/

Reply:
> Do other CLI tools consider their stdout/stderr as semver-protected if its meant for human readability?

It depends on the output, if it's meant to be human readable, then usually no. All for omitting this since this is usually not assumed to be an API.

Reply:
Aligned, let's omit.